wdpk/docker/install.sh

```diff
@@ -21,13 +21,15 @@ ARCH="$(uname -m)"
 
 # download docker binaries
 cd "${APKG_PATH}"
-TARBALL="docker-18.06.0-ce.tgz"
+TARBALL="docker-18.09.1.tgz"
 
 if [ ${ARCH} != "x86_64" ]; then
     ARCH="armel"
     # JediNite provides custom docker packages for ARM
     # They are based on docker-runc without seccomp, as the kernel doesn't support it
-    wget "https://github.com/JediNite/docker-ce-WDEX4100-binaries/raw/master/armv7l-WDEX4100/${TARBALL}" --no-check-certificate
+    # The latest version is not available yet
+    # wget "https://github.com/JediNite/docker-ce-WDEX4100-binaries/raw/master/armv7l-WDEX4100/${TARBALL}" --no-check-certificate
```

Comment:
Versioning gets a bit ugly here... and changing the tarball name breaks the installation for arm platforms. The variable is used further in the code too.

Reply:
Well, the arm package is available on the main repo
https://download.docker.com/linux/static/stable/armhf/docker-18.09.1.tgz
I haven't check what is added on Jedinite repo, but it works on my RPI. I don't have any arm nas

Reply:
Docker 18.09 is broken on arm6, but works on arm6
https://github.com/moby/moby/issues/38175#issuecomment-453726361

Reply:
JediNite had to rebuild docker without the seccomp module. 
I'm not an expert on the field though...